static/js/server_events_dispatch.js

```diff
@@ -74,7 +74,7 @@ exports.dispatch_normal_event = function dispatch_normal_event(event) {
             page_params.realm_email_changes_disabled = event.value;
             settings_org.toggle_email_change_display();
         } else if (event.op === 'update' && event.property === 'add_emoji_by_admins_only') {
-            page_params.add_emoji_by_admins_only = event.value;
+            page_params.realm_add_emoji_by_admins_only = event.value;
```

Comment:
@sarahcstringer I guess this is a nice example of a bug that would stop being possible when we add `property_types` to this block :)